CSE-Hub/user/profile.php

```diff
@@ -77,13 +119,31 @@
             <br/>
             submissions
             <br/>
-            pi chart of Submissions
-            and... space to represent other user stats
+            <div id="chart_div"></div>
+            <br><br>
         </section>
     </section>
 
     <footer>
         <p>This is the footer element.</p>
     </footer>
 </body>
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+<script src="https://cdn.anychart.com/js/8.0.1/anychart-core.min.js"></script>
+<script>
+google.charts.load('current', {'packages':['corechart']});
+    google.charts.setOnLoadCallback(draw_my_chart);
+    function draw_my_chart() {
+        var data = new google.visualization.DataTable();
+        data.addColumn('string', 'language');
+        data.addColumn('number', 'Nos');
+		for(i = 0; i < my_2d.length; i++)
+    data.addRow([my_2d[i][0], parseInt(my_2d[i][1])]);
+    var options = {title:'stats',
+                       width:600,
+                       height:500};
+    var chart = new google.visualization.PieChart(document.getElementById('chart_div'));
+    chart.draw(data, options);
+    }
```

Comment:
How about adding some resources to learn these in [Resources.md](https://github.com/harshraj22/contest/blob/develop/resources/resources.md) ?

Reply:
I will add...